src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/removal-under-maintenance/page.tsx

```diff
@@ -22,6 +25,8 @@ export default async function RemovalUnderMaintenance() {
   const countryCode = getCountryCode(headers());
   const profileId = await getOnerepProfileId(session.user.subscriber.id);
   const latestScan = await getLatestOnerepScanResults(profileId);
+  // TODO: remove after testing
+  await getScanResultsWithBrokerUnderMaintenance(profileId);
```

Comment:
Don't forget to remove before merging :)

Reply:
```suggestion
```
Del before merging